source/administration/changelog.md

```diff
@@ -404,6 +404,9 @@ Multiple setting options were added to `config.json`. Below is a list of the add
 
 ## Release v5.25 - [ESR](https://docs.mattermost.com/administration/release-definitions.html#extended-support-release-esr)
 
+- **v5.25.6, released 2020-11-10**
+  - Fixed an issue where the Compliance Exports were taking too long on large deployments. This was fixed with a performance optimization of the message export query.
+  - Bumped up Go patch version to fix an issue where a possible livelock was detected in the app server under heavy load. [MM-26584](https://mattermost.atlassian.net/browse/MM-26584)
```

Comment:
Need confirmation from Agniva on whether more details are needed / whether this text should be changed.

Reply:
Yes, this is fine. Maybe also mention the patch version - which is 1.14.6.

Reply:
Thanks, added.